homeassistant/components/logbook/__init__.py

```diff
@@ -377,29 +377,62 @@ def json_events() -> web.Response:
 
 
 def _humanify(
-    hass: HomeAssistant,
     rows: Generator[Row, None, None],
+    entities_filter: EntityFilter | Callable[[str], bool] | None,
+    registry: er.EntityRegistry,
```

Comment:
Let's rename this to `ent_reg` to better indicate what registry it is.

Reply:
Good call 👍